server/core/config/raw/jobs.go

```diff
@@ -0,0 +1,89 @@
+package raw
+
+import (
+	"regexp"
+
+	validation "github.com/go-ozzo/ozzo-validation"
+	"github.com/pkg/errors"
+	"github.com/runatlantis/atlantis/server/core/config/valid"
+)
+
+/*
+Ref: https://docs.aws.amazon.com/AmazonS3/latest/userguide/bucketnamingrules.html
+S3 bucket naming rules:
+1. Must be between 3 - 63 characters long
+2. Must be lowercase letters, numbers, dots and hyphens.
+3. Must begin and end with a letter or number.
+*/
+const ValidBucketNameRegEx = `^[a-z0-9-.]*$`
+const ValidStartCharacterRegEx = `^[a-z0-9][a-z0-9-.]*$`
+const ValidEndCharacterRegEx = `^[a-z0-9-.]*[a-z0-9]$`
```

Comment:
I understand the sentiment here, but not sure I really see the value of this.

These naming rules are for creating buckets but the bucket should already be created when someone is passing it as config to atlantis.  If the bucket fails validation because of these rules, it only really tells the user that maybe they mispelt their bucket name, but nothing else.

Also, consider aws changes its naming rules and we prevent someone from passing their bucket in here even though its valid in AWS but not here?

Reply:
Makes sense! Maybe we should just have a basic character validation (lowercase letters, digits, hyphen and dot) and leave out the start and end character validation.  I doubt AWS would change the valid characters for bucket names! Or do you think we should remove the validation entirely?

Reply:
I dont think the validation is really useful at all given that we aren't validating creation of a bucket. 

If anything useful validation would be something like checking the bucket exists in AWS, but i dont think we need to do that.

Reply:
Gotcha! I'll remove the validation then.